projects/kit/components/input-number/test/input-number.component.spec.ts

```diff
@@ -271,6 +271,21 @@ describe('InputNumber', () => {
         });
     });
 
+    it(`Formats value with precision > 6 if the element out of focus`, () => {
+        const value = '0,0000000001';
```

Comment:
please, also add test where you put `5.5e-10` value (as example)

Reply:
I added more cases with different values (but I haven't used exponent record, because user can't enter value with e notation)